src/Arcus.Scripting.Tests.Integration/appsettings.json

```diff
@@ -18,6 +18,9 @@
     "IntegrationAccount": {
       "Name": "#{Arcus.IntegrationAccount.Name}#"
     },
+    "AppService": {
+      "Name": "#{Arcus.AppService.Name}#"
```

Comment:
You found everything on Azure and Azure DevOps? No problems?

Reply:
Yes no problems, had access to everything I needed 👍🏻